.github/workflows/uffizzi-preview.yml

```diff
@@ -0,0 +1,84 @@
+name: Deploy Uffizzi Preview
+
+on:
+  workflow_run:
+    workflows:
+      - "Build PR Image"
+    types:
+      - completed
+
+
+jobs:
+  cache-compose-file:
+    name: Cache Compose File
+    runs-on: ubuntu-latest
+    outputs:
+      compose-file-cache-key: ${{ env.COMPOSE_FILE_HASH }}
+      pr-number: ${{ env.PR_NUMBER }}
+    steps:
+      - name: 'Download artifacts'
```

Comment:
Mostly curious: why do we need two separate workflows given that one always runs after the other? If it was a single workflow, we could cache the compose file right after it is rendered, without all the artifact logic, or is there a catch?